src/generateDocumentation.js

```diff
@@ -292,12 +292,33 @@ function generateModels(templateModels) {
 
   templateModels.forEach((templateModel) => {
     if (!templateModel.name || !templateModel.schema) return;
-    models[templateModel.name] = templateModel.schema;
+    models[templateModel.name] = generateModelSchema(templateModel.schema);
   });
 
   return models;
 }
 
+function generateModelSchema(templateSchema) {
+  if (!templateSchema) return templateSchema;
```

Comment:
I know I know, but take in mind that this line is meant only as a guard/safe-check to avoid executing the below code when unneeded. Besides, I thought (and still think) better not to change the original model schema, except for replacing the `$ref` field when needed.

I mean, if the passed model is null, or a falsy field (e.g., `"default": ""`, `"minimum": 0` or `"nullable": false`), then we simply return it as-is. This way we keep the output documentation preview consistent with the input template.